XamarinCommunityToolkit/AttachedProperties/ProgressBarAttachedProperties.shared.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.AttachedProperties
```

Comment:
@jfversluis Should we put it inside "Views"?

Reply:
Well, this class looks like behavior (attached behavior)

Can we refactor it to be a real behavior?
Or we can do it later

Reply:
Let's leave it for now... these are 2 different concepts ( although similar in design ).